tests/pipeline_operations_test.py

```diff
@@ -140,21 +138,19 @@ def test_filter_by_key_nonempty_public_partitions(self, distributed):
         self.assertListEqual(result, [(11, (1, 11, 111))])
 
     def test_sample_fixed_per_key(self):
-        spark_operations = SparkRDDOperations()
         data = [(1, 11), (2, 22), (3, 33), (1, 14), (2, 25), (1, 16)]
         dist_data = SparkRDDOperationsTest.sc.parallelize(data)
-        rdd = spark_operations.sample_fixed_per_key(dist_data, 2)
+        rdd = self.ops.sample_fixed_per_key(dist_data, 2)
         result = dict(rdd.collect())
         self.assertEqual(len(result[1]), 2)
         self.assertTrue(set(result[1]).issubset({11, 14, 16}))
         self.assertSetEqual(set(result[2]), {22, 25})
         self.assertSetEqual(set(result[3]), {33})
 
     def test_count_per_element(self):
-        spark_operations = SparkRDDOperations()
         data = ['a', 'b', 'a']
         dist_data = SparkRDDOperationsTest.sc.parallelize(data)
-        rdd = spark_operations.count_per_element(dist_data)
+        rdd = self.ops.count_per_element(dist_data)
```

Comment:
Could you also `replace spark_operations` with `self.ops` in the following method `test_reduce_accumulators_per_key`?
I couldn't figure out how to make a comment to a line which wasn't added by you. :-)

Reply:
Thanks, fixed. I don't know a way to comment on not-changed lines either :)